spec/models/user_spec.rb

```diff
@@ -2905,5 +2905,18 @@ def reset_last_seen_cache!(user)
       expect(moderator.last_seen_reviewable_id).to eq(mod_reviewable.id)
       expect(user.last_seen_reviewable_id).to eq(group_reviewable.id)
     end
+
+    it "publishes a message to the user's /reviewable_counts message bus channel" do
+      user.update!(admin: true)
+      reviewable = Fabricate(:reviewable)
+      messages = MessageBus.track_publish do
+        user.bump_last_seen_reviewable!
+      end
+      expect(messages.size).to eq(1)
+      message = messages[0]
+      expect(message.channel).to eq("/reviewable_counts/#{user.id}")
```

Comment:
I don’t know if you’re aware of this matcher but we could rewrite these expectations as:
```ruby
      expect(messages.first).to have_attributes channel: "/reviewable_counts/#{user.id}",
                                                user_ids: [user.id],
                                                data: { unseen_reviewable_count: 0 }
```

Reply:
done!